source/common/stats/symbol_table_impl.h

```diff
@@ -400,6 +407,29 @@ class StatNameManagedStorage : public StatNameStorage {
   SymbolTable& symbol_table_;
 };
 
+class StatNameManagedContainer {
```

Comment:
Could you add a docstring with an example usage of StatNameManagedContainer, where it fits into the stat name machinery?

Reply:
done